ecfo/readme.md

```diff
@@ -0,0 +1,29 @@
+
+
+# /rains
+This [W3ID](https://w3id.org) provides a persistent URI namespace for the Emission Conversion Factors Ontology (ECFO).
+
+
+Redirects: 
+
+https://eats-uoa.github.io/ECFO-Emission-conversion-Factor-Ontology/index-en.html
+
+https://eats-uoa.github.io/ECFO-Emission-conversion-Factor-Ontology/ecfo.json
+
+https://eats-uoa.github.io/ECFO-Emission-conversion-Factor-Ontology/ecfo.nt
+
+https://eats-uoa.github.io/ECFO-Emission-conversion-Factor-Ontology/ecfo.xml
+
+https://eats-uoa.github.io/ECFO-Emission-conversion-Factor-Ontology/ecfo.ttl
+
+## Uses
+The ontology is aimed at providing a  vocabulary for describing emission conversion factors that are used in the calculations of carbon footprint.  
+
+## Contact
+This space is administered by:  
+
+**Milan Markovic**  ,
+GIthub ID: m-markovic,
+https://www.abdn.ac.uk/ncs/profiles/milan.markovic/,
```

Comment:
To preserve the line breaks as appears to be desired, each of these lines should end with either 2 space characters, or 1 `<br>`.